package.json

```diff
@@ -17,87 +17,87 @@
     "test:unit": "vue-cli-service test:unit"
   },
   "dependencies": {
-    "@lumino/algorithm": "^1.3.2",
-    "@lumino/datagrid": "^0.3.1",
-    "@lumino/default-theme": "^0.1.8",
-    "@lumino/widgets": "^1.9.3",
-    "apollo-boost": "^0.4.4",
-    "apollo-utilities": "^1.3.2",
+    "@lumino/algorithm": "^1.3.3",
+    "@lumino/datagrid": "^0.12.0",
+    "@lumino/default-theme": "^0.4.4",
+    "@lumino/widgets": "^1.13.4",
+    "apollo-boost": "^0.4.9",
```

Comment:
When we first added Apollo, we used `apollo-boost`. Which is just the `apollo-client` wrapped by several utility functions, and features like caching.

Later, after realizing we didn't need any of the features in `apollo-boost`, I started moving the `WorkflowService` to use just `ApolloClient`. That also became necessary as we started customizing the client for the WebSockets.

I will have to confirm, but I believe I have finished getting rid of `apollo-boost`, but **added the dependencies for the non-apollo-boost setting in the `devDependencies`**.

If so, we can remove the Apollo dependencies from `devDependencies`, move them up here, and maybe reduce our list of dependencies, as ApolloBoost includes more code/dependencies.